.appveyor.yml

```diff
@@ -4,6 +4,8 @@ clone_folder: C:\MuseScore
 # set clone depth
 clone_depth: 3                      # clone entire repository history if not defined
 
+os: Visual Studio 2013
```

Comment:
Could also be `os: Previous Visual Studio 2015` I believe?

Reply:
Why? AppVeyor removed Qt 5.4 from VS2015 image

Reply:
Because Qt 5.4 is EOL